KarhooUISDK/CommonUI/Buttons/ItemFilterButton.swift

```diff
@@ -80,7 +80,7 @@ class ItemButton: UIButton {
     
     private func updateSelectedDesign() {
         let borderColor = isSelected ? KarhooUI.colors.accent.cgColor : UIColor.clear.cgColor
-        let color = isSelected ? KarhooUI.colors.lightAccent : KarhooUI.colors.background2
+        let color = isSelected ? KarhooUI.colors.lightAccent : KarhooUI.colors.border
```

Comment:
I don't like that kind of solutions. We are using color.border for a background color. Shouldn't we in that case create new background color (with the same values as border) and use it when we need this for background?

Reply:
This is something I carried over while cherrypicking your branch for the lint errors, along with the hex color updates. I can discard this change if it wasn't intended, but if that's the case we'll need it modified in the passenger details cell as well. LMK which is the correct value for this.